.github/workflows/build.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v2
         with:
-          java-version: '8'
+          java-version: '11.0.15'
```

Comment:
what if we just make this change - so that we are building with v11 but still targeting v8? Then we could upgrade other infra to run v11 too.

Reply:
@snopoke That would mean we set `sourceCompatibility` to Java 11 while use the `targetCompatibility` as Java 8 ? I thought about that though it doesn't seem like a [good practice](https://stackoverflow.com/a/22326989/3811963) to have source and target different. Also from [this thread](https://discuss.gradle.org/t/why-cant-i-use-different-sourcecompatibility-targetcompatibility-with-hello-world/11958/2) it seems like these comptibilty settings are used to do the opposite i.e. in our case update infra to Java 11 first and have code still compiles with Java 8 but target Java 11.

Reply:
OK, I didn't know you could do that (compile with v8 but target v11)